scripts/create_policy

```diff
@@ -160,11 +189,183 @@ def process_signature_verification_keys(verification_keys, policy):
     return policy
 
 
+def analyze_rpm_pkg(pkg):
+    """Analyze a single RPM package."""
+    ts = rpm.TransactionSet()
+    ts.setVSFlags(rpm.RPMVSF_MASK_NOSIGNATURES | rpm.RPMVSF_MASK_NODIGESTS)
+
+    with open(pkg, "rb") as f:
+        hdr = ts.hdrFromFdno(f)
+    # TODO - How are symbolic links managed with IMA
```

Comment:
if `ls` is a symbolic link to `busybox` then busybox will be logged an measured

Reply:
Thanks, that simplify the problem. I will update the doc.